src/Tokenizer.js

```diff
@@ -585,7 +585,7 @@ module.exports = class Tokenizer {
       return {
         type: 'del',
         raw: cap[0],
-        text: cap[1]
+        text: cap[2]
```

Comment:
I had to change to capture positions on the `del` regex. I know we have run into this before where users might be using those capture positions so this could be a breaking change.

Reply:
Everything looks good to me on a first lookthrough, except as you say this might be a breaking change.

Reply:
Would it be better to release v2 with this change or should we hold off on this until we have a more important breaking change (or just more breaking changes)?

Reply:
As far as breaking changes go, this one seems pretty mild to me. It only affects users who have tinkered with extending Marked in some way (directly accessing the tokenizer, etc.), and specifically only the `del` marker, which I suspect is rare. Any user capable of building such extensions in their own code should likely have minimal issues accommodating this change as long as they are aware the change happened.

So... breaking change? Yeah, but only for special case users. By opening up Marked to be extendable (by literally overwriting the code for the Tokenizer\Lexer\Renderer, etc.,) there are probably more of these types of breaking changes than not, because any change to the parsing pipline could affect someone's custom extension in a way we can't predict.  Do we count every change to the pipeline as "breaking" and wait until a major version change because technically a user might be accessing the Lexer in a way we didn't expect? It's a tough question.

Reply:
Ya [here](https://github.com/markedjs/marked/issues/1073) is one of the discussions we had a while ago about  changing capture position as a breaking change.

While it is possible for someone to write a tokenizer function that still uses the regexes from rules.js, it is not documented anywhere or explicitly made public. So maybe this should not bump the major version but list it as a possible breaking change in the release note?

Reply:
Right... AFAIK, Semver only needs to change the major version if the *public* API is changed? By the time a user is delving into the internals of Marked to make a custom Tokenizer, that no longer seems to be public anymore, which would make it fair game to update.

Reply:
How about semver minor?

Reply:
It isn't a new feature. I think a patch version update with a note in the release notes about the possible breaking change.